### DISCUSSION TITLE
umi/max的菜单如何使用自定义 icon 而非 @ant-design/icons ？

### DISCUSSION THREAD

**Discussion by xliez** [umijs/umi#11874] [Q&A] [answered]
目前是不是只支持 @ant-design/icons 自带的图标，可以自定义吗？

**Comment by fz6m** [comment, marked as answer]
如果非要自定义，可以参考 `./src/.umi/plugin-layout/runtime.tsx` 里面的 [`patchRoutes`](https://umijs.org/docs/api/runtime-config#patchroutes-routes-) 部分，这是运行时函数，可以参考写在 `./src/app.ts` 里，自己 patch 对应路由的 icon 为一个想要的 react 节点。
